src/org.ts

```diff
@@ -68,7 +77,7 @@ export async function createOrg(
   options.logger.info({ name: orgData.name }, 'Org created');
   return {
     didSucceed: true,
-    result: { name: org.name },
+    result: { name: orgData.name },
```

Comment:
Why though? The DB shouldn't be transforming the org name in any way (if it did, that'd be a bug).

Note that line 77 was already using `orgData.name` instead of the original `org.name`, so part of the rationale for this change was consistency.

The main reason for making this change, though, is that the argument to `orgModel.create()` is now more complicated as it contains two new fields (`privateKeyRef` and `publicKey`) -- both of which need to be computed. I didn't want to do of all that complicated stuff between the `try`/`catch`, so I decided to assign it to another variable (`org`) outside the `try`/`catch`.

So if we stick to the original approach of reading the output of `orgModel.create(...)`, we're going to end up with two instances of the same org (`inputOrg` and `outputOrg`, for example)... Even though we don't really need two instances of the same org.